lib/path.js

```diff
@@ -39,16 +45,29 @@ const isWsp = (c) => {
   );
 };
 
+/**
+ * @param {string} c
+ */
 const isDigit = (c) => {
   const codePoint = c.codePointAt(0);
+  if (codePoint === undefined) {
```

Comment:
I'd prefer `codePoint == null`

Reply:
Ok, done. But IMO it makes the code much harder to reason about when one relies on auto-conversion and `==` checks. For a developer is quite baffling why if the method can return `undefined` you are checking for null.

Reply:
`value == null` checks for nullable. This makes less painful dilemma whether use null or undefined for not existing value and works as interop for existing api.

Reply:
Well in this case the API was already written and is from the standard library. This has nothing to do with interoperability for the existing API.